substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/jdk/JmxCommonFeature.java

```diff
@@ -207,6 +207,7 @@ private static void configureSerialization(BeforeAnalysisAccess access) {
                         "[B", "com.oracle.svm.core.jdk.UnsupportedFeatureError",
                         "java.io.IOException", "java.lang.Boolean", "java.lang.ClassCastException", "java.lang.Error",
                         "java.lang.Exception", "java.lang.IllegalArgumentException", "java.lang.IllegalStateException",
+                        "java.lang.Double",
```

Comment:
Should we, for consistency, also add `[Ljava.lang.Double;`, and other `java.lang` base types like `Character` and `Float` (or are these never used in JMX)?

Reply:
It doesn't look like any of the platform mxbean attributes use them, but it's possible that some user's custom standard mbeans may use them. So we might as well add them to the serialization config too.

Reply:
If that's the case, I'd argue that the user also needs to provide custom serialization config. Could you just revert to the previous version again?

Reply:
ok will do